### ISSUE TITLE
Malformed tar.gz files in replication folder

### ISSUE THREAD

**Issue by carolemieux** [microsoft/codamosa#13]
One user reports that `benchmarks-docker.tar.gz` is malformed: confirmed md5sum is different than on my machine. I will investigate.

If anyone else has the same issue in the meantime, here is a backup [benchmarks-docker.tar.gz](https://drive.google.com/file/d/10tL4zZcZthhLwURsqmCProVT6QrdT2ed/view?usp=drive_link) and [codamosa-docker.tar.gz](https://drive.google.com/file/d/1N5XtaAB2XgXHTJahIPjjUzCjOJ-YWl6n/view?usp=drive_link)

**Comment by khangtran2020** [comment]
When will this be resolve ?

**Comment by carolemieux** [comment]
I believe the issue may be coming from the fact that the docker images are stored on git-lfs. I cloned the repo without lfs installed and saw that the images were broken. However, upon installing git-lfs (https://docs.github.com/en/repositories/working-with-files/managing-large-files/installing-git-large-file-storage), and re-cloning the repo, I was able to load the images from the .tar.gz correctly.

If we can confirm this fixes the issue, I will update the documentation accordingly.
